src/pages/index.js

```diff
@@ -167,12 +284,26 @@ previewModalClose.addEventListener("click", () => {
   closeModal(previewModal);
 });
 
-modalEditForm.addEventListener("submit", handleProfileFormSubmit);
-modalCardForm.addEventListener("submit", handleCardFormSubmit);
+avatarModalButton.addEventListener("click", () => {
+  openModal(avatarModal);
+});
 
-initialCards.forEach((cardItem) => {
-  const cardElement = getCardElement(cardItem);
-  cardsList.prepend(cardElement);
+avatarCloseBtn.addEventListener("click", () => {
```

Comment:
**Could be improved**

You can make a universal handler for any close buttons.

It will look something like this:

```js
// Find all close buttons
const closeButtons = document.querySelectorAll('.modal__close');

closeButtons.forEach((button) => {
  // Find the closest popup only once
  const popup = button.closest('.modal');
  // Set the listener
  button.addEventListener('click', () => closePopup(popup));
});



```

That’s why we make universal css classes for similar elements: here it’s `modal__close` for any close button.

So now you can add 100 modals: their close buttons will be handled automatically.